bCNC/lib/tkExtra.py

```diff
@@ -3075,13 +3076,18 @@ def afterHide(self):
 	# Public methods
 	# ----------------------------------------------------------------------
 	def get(self, first=None, last=None):
+		PY2 = sys.version_info[0] == 2
```

Comment:
i would prefer python version checks not being used in code. is there another way to solve this? like better use of isinstance() ?

Reply:
Thanks @Harvie checkout commit 723bb86 please